README.md

```diff
@@ -73,7 +73,10 @@ Links to the Demand API documentation are included for each function.
 [Launch Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem-launch): launch_line_item(project_id, line_item_id)  
 [Pause Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem-pause): pause_line_item(project_id, line_item_id)  
 [Update Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem): update_line_item(project_id, line_item_id, line_item_data)  
-[Get Line Item Detailed Report](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/get-detailed-line-item): get_line_item_detailed_report(project_id, line_item_id)  
+[Get Line Item Detailed Report](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/get-detailed-line-item): get_line_item_detailed_report(project_id, line_item_id) 
+
+### todo: will update this once it is published in developers.dynata.com
```

Comment:
Not sure this note needs to be in the code. Why not denote it in Github and @DynataRidley will make sure to wait until the green light to release the next version to PyPi?

Reply:
@DynataRidley , I have updated with the changes to README. Could you kindly review.